frontend/src/app/main/ui/workspace/sidebar/options/menus/component.cljs

```diff
@@ -839,13 +838,14 @@
                                     :on-close on-menu-close
                                     :menu-entries menu-entries
                                     :main-instance true}]])]
-        [:*
-         (for [[pos property] (map vector (range) properties)]
-           (let [val (str/join ", " (:values property))]
-             [:div {:key (str (:id shape) (:name property)) :class (stl/css :variant-property-row)}
-              [:> input-with-values* {:name (:name property) :values val :on-blur (partial update-property-name pos)}]
-              [:> icon-button* {:variant "ghost"
-                                :aria-label (tr "workspace.shape.menu.remove-variant-property")
-                                :on-click (partial remove-property pos)
-                                :icon "remove"
-                                :disabled (<= (count properties) 1)}]]))]]])))
+        (when-not multi
+          [:*
+           (for [[pos property] (map vector (range) properties)]
+             (let [val (str/join ", " (:values property))]
+               [:div {:key (str (:id shape) (:name property)) :class (stl/css :variant-property-row)}
+                [:> input-with-values* {:name (:name property) :values val :on-blur (partial update-property-name pos)}]
+                [:> icon-button* {:variant "ghost"
+                                  :aria-label (tr "workspace.shape.menu.remove-variant-property")
+                                  :on-click (partial remove-property pos)
```

Comment:
This partial creates a new callback on each render. Should be avoided. There are approaches that I have explained  in several sessions on how to handle this cases.